extensions/emoji/extend.php

```diff
@@ -30,4 +33,7 @@
         }),
 
     new Extend\Locales(__DIR__.'/locale'),
+
+    (new Extend\Settings)
+        ->serializeToForum('flarum-emoji.cdn', 'flarum-emoji.cdn'),
```

Comment:
let's drop the `cdn` const from `cdn.js` and instead add a default value here with that value `->default(key, value)`

Reply:
Ok,I will process it a few days later, have a nice weekend^_^